python/pyarrow/tests/test_dataset.py

```diff
@@ -4615,6 +4624,13 @@ def test_write_iterable(tempdir):
     result = ds.dataset(base_dir, format="ipc").to_table()
     assert result.equals(table)
 
+    base_dir = tempdir / 'inmemory_pycapsule'
+    stream = TableStreamWrapper(table)
+    ds.write_dataset(stream, base_dir,
+                     basename_template='dat_{i}.arrow', format="feather")
```

Comment:
Can we use `ipc` not `feature` for `.arrow` file?

```suggestion
                     basename_template='dat_{i}.arrow', format="ipc")
```